events/app_watcher_test.go

```diff
@@ -70,62 +72,79 @@ var _ = Describe("AppWatcher", func() {
 
 	Describe("Run", func() {
 		It("Registers metrics on startup", func() {
-			// go appWatcher.Run()
-			// defer appWatcher.Close()
+			defer appWatcher.Close()
```

Comment:
Now that Run() is started in BeforeEach in NewAppWatcher, I think all the Close() calls could be moved to AfterEach, without `defer`.  Resources should be cleaned up close to where they’re allocated - if you start a go routine in a test, close it in the test with `defer`. If you start it in BeforeEach, close it in AfterEach. 

This might need some careful thought for tests where we Close() in the test itself though.